pkgs/development/python-modules/pythonix/default.nix

```diff
@@ -18,6 +18,10 @@ buildPythonPackage rec {
 
   buildInputs = [ nix boost ];
 
+  postInstall = ''
+    PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH python -c "import nix"
```

Comment:
Afaik the same as: 

```
pythonImportsCheck = [ "nix" ];
```

Reply:
Yeah I had tried that but oddly I get this error:
```
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: nix
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/m7080pw0ryjk0jhljp55rq1hd2qy8gki-python3-3.8.6/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'nix'
```
And yet the manual check works, as well as the resulting package in a nix-shell. Maybe pythonImportsCheck doesn't work with the meson build?

Reply:
Mhm. Maybe I miss some python metadata that importlib expects?